tests/unit/test_param_expand.py

```diff
@@ -154,9 +154,9 @@ def test_graph_expand_offset_1(self):
         """Test graph expansion with a -ve offset."""
         self.assertEqual(
             self.graph_expander.expand("bar<i-1,j>=>baz<i,j>"),
-            set(["baz_i0_j0",
-                 "baz_i0_j1",
-                 "baz_i0_j2",
+            set(["bar_i-32768_j0=>baz_i0_j0",
```

Comment:
Yikes!

Does this out of bounds parameter get cleared out later?

Reply:
It does. From what I remember of the algorithm, in the first pass it will expand the graph with these values. But later it removes them.